packages/vapor/scss/controls/numeric-input.scss

```diff
@@ -0,0 +1,13 @@
+.numeric-input {
+    .mod-max-2-digit {
+        width: $numeric-input-2-digit-width;
+    }
+
+    .mod-max-3-digit {
+        width: $numeric-input-3-digit-width;
+    }
+
+    .mod-max-4-digit {
+        width: $numeric-input-4-digit-width;
+    }
```

Comment:
I bet we could do a loop with a simple calculation (something like `padding + i * 10` )